name,value,relevant log,explanation
io.bytes.per.checksum,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
mapreduce.shuffle.pathcache.expire-after-access-minutes,5,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
dfs.client.socket.send.buffer.size,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
dfs.client.hedged.read.threadpool.size,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
mapreduce.fileoutputcommitter.algorithm.version,2,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
dfs.datanode.http.internal-proxy.port,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
io.bytes.per.checksum,0,"ERROR	Task: attempt_1701457076904_0394_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
mapreduce.shuffle.pathcache.expire-after-access-minutes,5,"ERROR	Task: attempt_1701457076904_0394_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.client.socket.send.buffer.size,0,"ERROR	Task: attempt_1701457076904_0394_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.client.hedged.read.threadpool.size,0,"ERROR	Task: attempt_1701457076904_0394_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
mapreduce.fileoutputcommitter.algorithm.version,2,"ERROR	Task: attempt_1701457076904_0394_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.datanode.http.internal-proxy.port,0,"ERROR	Task: attempt_1701457076904_0394_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
io.bytes.per.checksum,0,"INFO	Diagnostics report from attempt_1701457076904_0394_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
mapreduce.shuffle.pathcache.expire-after-access-minutes,5,"INFO	Diagnostics report from attempt_1701457076904_0394_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.client.socket.send.buffer.size,0,"INFO	Diagnostics report from attempt_1701457076904_0394_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.client.hedged.read.threadpool.size,0,"INFO	Diagnostics report from attempt_1701457076904_0394_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
mapreduce.fileoutputcommitter.algorithm.version,2,"INFO	Diagnostics report from attempt_1701457076904_0394_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.datanode.http.internal-proxy.port,0,"INFO	Diagnostics report from attempt_1701457076904_0394_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
mapreduce.shuffle.pathcache.expire-after-access-minutes,5,INFO	1 failures on node cf6e5ca2712c,value-hit
mapreduce.fileoutputcommitter.algorithm.version,2,INFO	1 failures on node cf6e5ca2712c,value-hit
io.bytes.per.checksum,0,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,0,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,0,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,0,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
